OpenTabletDriver.Plugin/Tablet/Wheel/IRelativeWheelReport.cs

```diff
@@ -0,0 +1,13 @@
+namespace OpenTabletDriver.Plugin.Tablet.Wheel
+{
+    /// <summary>
+    /// An auxiliary report containing states of a wheel/ring input.
+    /// </summary>
+    public interface IRelativeWheelReport : IDeviceReport
+    {
+        /// <summary>
+        /// The relative wheel movement from its previous state, or null to indicate an absence of touch.
+        /// </summary>
+        int WheelDelta { get; set; }
```

Comment:
```suggestion
        int? WheelDelta { get; set; }
```

To distinguish between a stationary touch and the absence (removal really) of touch

Reply:
(context: https://github.com/OpenTabletDriver/OpenTabletDriver/pull/3497#discussion_r1846889119)

Quote by joha4270:
> I would instead suggest going back to `int?`.
> 
> I don't quite know why the original author originally went with a uint, but at least the HS610 can also detect that the finger is touching the wheel, even if its stationary.
> 
> So a report of zero would be the first touch before movement, and a report of null would indicate that the touch has ended.

which I agree with